File: app/src/main/java/remix/myplayer/bean/kugou/KSearchResponse.java
package remix.myplayer.bean.kugou;

import java.io.Serializable;
import java.util.List;

/**
 * Created by Remix on 2017/11/21.
 */

public class KSearchResponse implements Serializable {
    private static final long serialVersionUID = -4932979292293561928L;
    public String info;
    public int status;
    public String proposal;
    public String keyword;
    public List<SearchActualData> candidates;

    public static class SearchActualData implements Serializable {
        private static final long serialVersionUID = -2720016148055276583L;
        public String soundname;
        public int krctype;
        public String nickname;
        public String originame;
        public String accesskey;
        public String origiuid;
        public int score;
        public int hitlayer;
        public int duration;
        public String sounduid;
        public String song;
        public String uid;
        public String transuid;
        public String transname;
        public int adjust;
        public int id;
        public String singer;
        public String language;
    }
}

File: app/src/main/java/remix/myplayer/bean/kugou/KSearchResponse.kt
package remix.myplayer.bean.kugou

/**
 * Created by Remix on 2017/11/21.
 */

data class KSearchResponse(var info: String? = null,
                           val status: Int = 0,
                           val proposal: String? = null,
                           val keyword: String? = null,
                           val candidates: List<SearchActualData>) {
    data class SearchActualData(
            var soundname: String? = null,
            var krctype: Int = 0,
            var nickname: String? = null,
            var originame: String? = null,
            var accesskey: String? = null,
            var origiuid: String? = null,
            var score: Int = 0,
            var hitlayer: Int = 0,
            var duration: Int = 0,
            var sounduid: String? = null,
            var song: String? = null,
            var uid: String? = null,
            var transuid: String? = null,
            var transname: String? = null,
            var adjust: Int = 0,
            var id: Int = 0,
            var singer: String? = null,
            var language: String? = null
    )
}
